python/nx-cugraph/nx_cugraph/classes/graph.py

```diff
@@ -702,6 +710,7 @@ def _get_plc_graph(
             renumber=False,
             do_expensive_check=False,
             vertices_array=self._node_ids,
+            drop_multi_edges=not self.is_multigraph(),
```

Comment:
Nice. All you did was setting a flag and no performance degradation

Reply:
Yes, it ended up being very straightforward.  I did notice a slight slowdown of <2% after additional runs, but still worth doing.